### DISCUSSION TITLE
v0.5.2-alpha

### DISCUSSION THREAD

**Discussion by bowring** [CIRDLES/Tripoli#239] [Show and tell]
Various fixes.  Thanks to all who contributed issues!

## What's Changed
* Fixed #235 and related by @bowring in https://github.com/CIRDLES/Tripoli/pull/237
* 15may2024 by @bowring in https://github.com/CIRDLES/Tripoli/pull/238

**Full Changelog**: https://github.com/CIRDLES/Tripoli/compare/v0.5.1-alpha...v0.5.2-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/CIRDLES/Tripoli/releases/tag/v0.5.2-alpha'>v0.5.2-alpha</a>.</em>

**Comment by noahmclean** [comment]
Looking spiffier all the time!  Cycle mode is starting to look pretty good.  I'll contribute some issues on block mode stats and outputs.

**Comment by CaroleLafontaine** [comment]
Good afternoon Jim,
I tried to install and run the OG-Tripoli V0.5.2-alpha jar file.
But when I double-click the downloaded file to launched it, I obtain the following message text box.

![Java error message](https://github.com/CIRDLES/Tripoli/assets/44178443/07312cdd-9486-4ea1-a137-78a571942b69)
 What version of Java is required to run the new version of Tripoli?  
The current version I have currently installed is as follows :
![Actual version of Java installed](https://github.com/CIRDLES/Tripoli/assets/44178443/773a920f-1694-4649-a974-e1e195f85a2b)

Many thanks in advance for your help and guidance.

**Comment by bowring** [comment]
read the readme : https://github.com/CIRDLES/Tripoli/blob/main/README.md - it tells you

(btw, OG Tripoli is the old Tripoli)

**Comment by bowring** [comment]
also take a look at some DRAFT documentation we are working on: https://noahmclean.github.io/documentationTest/

**Reply by CaroleLafontaine** [reply]
Thank you very much Jim!

**Reply by noahmclean** [reply]
Please note that I moved the documentation to https://noahmclean.github.io/TripoliDocs/ as we work to get the documentation into the CIRDLES organization on GitHub without blowing up the existing CIRDLES website.  #HowISpentMySummerVacation

**Comment by noahmclean** [comment]
Hi Carol, 

Thanks for your interest in Tripoli!  I'm working on some more detailed documentation for Tripoli now.  You can find the Java download and install documentation here:

https://noahmclean.github.io/documentationTest/installation

Let me know if you have any problems!  I'm working now on some instructions for loading and plotting data.

**Reply by CaroleLafontaine** [reply]
Thank you Noah!  I will have a look at the documentation and let you know in need.
At the moment, we are using the latest Tripoli version 4.10.0.3, and ET_Redux V3.7.1 to process U-Pb data files.  
I wonder while installing the alpha version of OG-Tripoli for test purposes could cause problems to still using Tripoli v4.10.0.3?
Maybe I am to fare ahead and that is explained in the documentation!!

**Reply by noahmclean** [reply]
We're using both versions on our mass spec computer without problems at the moment.  If you run into problems, you can always blame them on Jim!

**Reply by ryanickert** [reply]
when using Redux and Tripoli, I tell the students to credit jim but blame noah

**Reply by noahmclean** [reply]
This is 100% correct. 🤣

**Comment by bowring** [comment]
for the record, version 4.10.0.3 is now known as "OGTripoli".  

the new Tripoli can run alongside the OGTripoli - in fact folks need to do this and compare results - they can both open the same data file at the same time - in both cases, just drag and drop!  They can both output files for ET_Redux - these should be compared too.  You can run two copies of ET_Redux as well - one for OGTripoli and one for Tripoli and compare results  This ought to be fun!  Plus you will find bugs - yeah!

**Comment by CaroleLafontaine** [comment]
UNCLASSIFIED - NON CLASSIFIÉ

Thank you for the notice Noah!
It is really appreciated since I've tried earlier and got an URL error 400.
